### PR TITLE
mapper can handle multi-value fields from export

### LINKED ISSUE #256: IMPORT: ensure multi-value and single fields
All fields are multi-value except for: 'orcid', 'embargo_date', 'file_name', 'distribution_license', according to the spreadsheet (https://docs.google.com/spreadsheets/d/1HZpXyVM8xaZZ9mE2uzsbUmWKbCd6NYJTESOrpQeMa18/)

### PR DESCRIPTION
**Pull request by jenlindner**
this commit ensures that the importer and mapper can ingest multi-value fields that occur in the format name1, name2, and corrects the bepress exported field name 'license' to 'distribution_license'.

Closes #256.

### CODE REVIEW

**Review by no-reply — COMMENTED** [review]
Looks good!

Did we confirm that this is the correct pattern with OHSU; if not do you want to hold the PR until then, or go ahead with it (I don't think it breaks any current behavior)?

File: spec/importer/mahonia_mapper_spec.rb
Comment (no-reply) [inline spec/importer/mahonia_mapper_spec.rb]: Do we need a ticket to ensure that `embargo_date` imports correctly and that embargoes lift properly after import?

**Comment by jenlindner** [comment]
We have asked but have't heard back yet. I'm fine with either scenario... the pattern might need to be adjusted but the returned data and tests would look the same, I think.

File: spec/importer/mahonia_mapper_spec.rb
Comment (no-reply) [reply spec/importer/mahonia_mapper_spec.rb]: cc: @mark-dce

**Review by no-reply — DISMISSED** [review]
:+1:

**Comment by no-reply** [comment]
@jenlindner indicated that she intends to make some changes before merging this, so I'm dismissing my review as stale. I'm happy to re-review at any time.

**Review by no-reply — DISMISSED** [review]
:robot:

**Comment by no-reply** [comment]
ughhh, looks like this needs conflicts resolved. @jenlindner can you rebase?

**Review by no-reply — APPROVED** [review]
:octocat: one more time.

**Comment by jenlindner** [comment]
thanks @no-reply :-)

**Pull request merged**
